community/kernel/src/main/java/org/neo4j/kernel/impl/store/StoreType.java

```diff
@@ -23,11 +23,12 @@
 
 import org.neo4j.graphdb.factory.GraphDatabaseSettings;
 import org.neo4j.kernel.impl.store.id.IdType;
+import org.neo4j.kernel.impl.storemigration.StoreFile;
 import org.neo4j.kernel.impl.store.counts.CountsTracker;
 
 public enum StoreType
 {
-    NODE_LABEL( StoreFactory.NODE_LABELS_STORE_NAME )
+    NODE_LABEL( StoreFile.NODE_LABEL_STORE )
```

Comment:
What's the point of this refactoring in general?
and why it's included in this PR?